src/utils.js

```diff
@@ -1,6 +1,6 @@
 import path from 'path-browserify';
 import mime from 'mime-types';
-import { deviceType, primaryInput } from 'detect-it';
+import {deviceType, primaryInput} from 'detect-it';
```

Comment:
Do we need to `npm install detect-it`? Also, it doesn't appear that we use `deviceType` and `primaryInput` below. Did you want to add it to the output of `getDeviceInfo()`?

Reply:
Interesting, I didn't actually import those functions to use in my getDeviceInfo() commit. Perhaps they were left over from another utility function?

Reply:
Oh I see, they were there before and your commit only changed the formatting. Thanks for clarifying. Do you think we should use any of the detect-it info in `getDeviceInfo` or do you think what you have already written covers all the bases?